app/build.gradle

```diff
@@ -103,7 +103,7 @@ ext.gleanGenerateMarkdownDocs = true
 ext.gleanDocsDirectory = "$rootDir/docs"
 
 android {
-    compileSdkVersion build_versions.target_sdk
+    compileSdkVersion build_versions.compile_sdk
```

Comment:
Shouldn't this be part of `defaultConfig`?

Reply:
Done.